includes/html/functions.inc.php

```diff
@@ -1449,6 +1449,34 @@ function get_arrays_with_mdadm($device, $app_id)
     return $arrays;
 }
 
+/**
+ * Get all seafile data from the collected
+ * rrd files.
+ *
+ * @param array $device device for which we get the rrd's
+ * @param int   $app_id application id on the device
+ * @param string  $category which category of seafile graphs are searched
+ * @return array list of seafile data
+ */
+function get_arrays_with_seafile($device, $app_id, $category)
```

Comment:
this function is almost a duplicate of get_arrays_with_mdadm(),  Can  you combine them?

Reply:
@murrant 
globalized function
which handles now mdadm. smart and seafile

Reply:
@murrant 
i combined it in a globalized function